python/paddle/fluid/optimizer.py

```diff
@@ -664,6 +664,113 @@ def _append_optimize_op(self, block, param_and_grad):
         return adadelta_op
 
 
+class RMSPropOptimizer(Optimizer):
+    """
+    Root Mean Squared Propagation (RMSProp) is an unpublished, adaptive learning
+    rate method. The original slides that proposed Rmsprop: Slide 29 of
```

Comment:
slides that proposed Rmsprop --> slides proposed Rmsprop

Reply:
Done.